resources/measures/upgrade_hvac_vrf_hr_doas/measure.rb

```diff
@@ -1333,6 +1400,23 @@ def run(model, runner, user_arguments)
       end
     end
 
+    # applicability: number of indoor units per outdoor unit
```

Comment:
change summary: adding applicability for total number of indoor units (<=41)

Reply:
In practice, would an engineer just adjust the zoning so there are more outdoor units to support the large number of indoor units?